sci-visualization/mantid/files/mantid-3.3.0-FindOpenCascade.patch

```diff
@@ -0,0 +1,84 @@
+diff --git a/Build/CMake/FindOpenCascade.cmake b/Build/CMake/FindOpenCascade.cmake
+index aefa011..346678d 100644
+--- a/Build/CMake/FindOpenCascade.cmake
++++ b/Build/CMake/FindOpenCascade.cmake
+@@ -4,13 +4,6 @@
+ # OPENCASCADE_FOUND If false, do not try to use OPENCASCADE
+ 
+ find_path ( OPENCASCADE_INCLUDE_DIR Standard.hxx PATHS
+-                /usr/include/opencascade
+-                /usr/include/oce
+-                /usr/local/include/opencascade
+-                /usr/local/include/oce
+-                /usr/local/inc
+-                /opt/OpenCASCADE/inc
+-                ${CMAKE_INCLUDE_PATH}/OpenCascade
+                 $ENV{CASROOT}/inc
+ )
+ 
+@@ -18,50 +11,56 @@ if ( WIN32 )
+   add_definitions ( -DWNT )
+ endif ( WIN32 )
+ 
+-set ( OC_REDHAT_RPM /opt/OpenCASCADE/lib64 )
++find_path ( OPENCASCADE_LIBRARY_DIR libTKernel.so PATHS
++               $ENV{CASROOT}/lib64
++)
++
+ find_library ( OPENCASCADE_LIB_TKERNEL
+                  NAMES TKernel
+-                 PATHS ${OC_REDHAT_RPM}         
++                 PATHS ${OPENCASCADE_LIBRARY_DIR}
```

Comment:
I see, the selected version lives in `${CASROOT}`, right? That could be used in the ebuild as well. This patch can be dropped by using cmake's `find_library` build-in options to add search locations using `-DCMAKE_LIBRARY_PATH=${CASROOT}` and `-DFIND_LIBRARY_USE_LIB64_PATHS=<ON/OFF>`. See <http://www.cmake.org/cmake/help/v3.0/command/find_library.html> for details.